profiles/arch/arm/package.use.mask

```diff
@@ -1,6 +1,11 @@
 # Copyright 1999-2017 Gentoo Foundation
 # Distributed under the terms of the GNU General Public License v2
 
+# Ilya Tumaykin <itumaykin+gentoo@gmail.com> (24 Sep 2017)
```

Comment:
This has to be done by Gentoo developers. I will add myself instead here. Hope you don't mind.

Reply:
I've been told before it's not necessary, i.e. I can put my name here because we have git history.
I don't mind though, do as you see fit :)